kratos/geometries/geometry.h

```diff
@@ -832,12 +832,72 @@ class Geometry : public PointerVector<TPointType>
         return normal;
     }
 
+    /**
+     * @brief It returns the vector, which is normal to its corresponding
+     *        geometry in the given integration point for the default
+     *        integration method.
+     * @param IntegrationPointIndex index in internal integration point list
+     * @return The normal in the given integration point
+     */
+    virtual array_1d<double, 3> Normal(
+        IndexType IntegrationPointIndex) const
```

Comment:
i would avoid this method and only use the following with a defaukt value

Reply:
The default is a member variable, thus I cannot put it as default.